sea-orm-codegen/src/entity/writer.rs

```diff
@@ -94,6 +95,12 @@ impl EntityWriter {
         self.entities
             .iter()
             .map(|entity| {
+                let entity_file = format!("{}.rs", entity.get_table_name_snake_case());
+                info!(
+                    "Generating {}",
+                    entity_file
+                );
```

Comment:
Is it possible to print the info of all table columns, one line per column.
https://github.com/SeaQL/sea-orm/blob/e00562695ebb99eca444acbce41aafd0c5f3d539/sea-orm-codegen/src/entity/base_entity.rs#L9

Reply:
I fixed in [this commit](https://github.com/SeaQL/sea-orm/pull/735/commits/b893a3c81356a0bfdc9678cc2ffb9e50a4c0290e).

If there is a cooler way to display, I will respond.